core/src/components/progress-bar/progress-bar.tsx

```diff
@@ -57,10 +57,35 @@ export class ProgressBar implements ComponentInterface {
    */
   @Prop({ reflect: true }) color?: Color;
 
+  /**
+   * Set to `"round"` for a tab-button with rounded corners, or `"rectangular"`
+   * for a tab-button without rounded corners.
+   *
+   * Defaults to `"round"` for the `ionic` theme, undefined for all other themes.
+   */
+  @Prop() shape?: 'round' | 'rectangular';
+
+  private getShape(): string | undefined {
+    const theme = getIonTheme(this);
+    const { shape } = this;
+
+    // TODO(ROU-11436): Remove theme check when shapes are defined for all themes.
```

Comment:
As mentioned in another PR, the Jira tickets within TODO needs to be accurate. Please clone ROU-11436 to track the work needed for progress bar.